web/src/app/users/UserContactMethodForm.js

```diff
@@ -34,6 +35,24 @@ export default class UserContactMethodForm extends React.PureComponent {
   }
 
   render() {
+    const locale = this.props.value
+
+    let exampleNumber
+
+    try {
+      exampleNumber = getExampleNumber(locale.countryCode)
+    } catch (e) {
+      exampleNumber = '+1 631 746 3748'
```

Comment:
We should use the international format for numbers as we're using that elsewhere

Reply:
Here's where we're formatting numbers for contact methods:
https://github.com/target/goalert/blob/master/graphql2/graphqlapp/contactmethod.go#L30

Here's an example of the different formats libphonenumber supports, for reference:
![image](https://user-images.githubusercontent.com/595010/70169830-3399ae00-1691-11ea-9697-ee21c5259c7f.png)